src/ShipCpanel.cpp

```diff
@@ -280,12 +287,12 @@ void ShipCpanel::Update()
 	int timeAccel = m_game->GetTimeAccel();
 	int requested = m_game->GetRequestedTimeAccel();
 
-	for (int i=0; i<6; i++) {
+	for (int i=0; i<7; i++) {
```

Comment:
I was considering using size of array, but decided to keep it close to the original (as sizeof()/sizeof() would make it muddled). Using enum value seems like a better idea than this although I'm a bit wary of this kind of things, because there is no way to ensure that given enum stays last or keeps its value. OTOH it would be better to declare the size of the array itself as something that isn't hardcoded and I definitely agree that magic numbers are bad so enum it is - I will make the change tomorrow.